src/verinfast/utils/utils.py

```
@@ -139,10 +139,28 @@ def chunks(lst, n):
         yield lst[i:i + n]
 
 
+# DebugLog
+# Usage: must pass in full path to a log
+# file as "file" OR pass in a path and the
+# class will create a default log file called
+# "log.txt" in the specified path
 class DebugLog:
-    def __init__(self, path: str, debug: bool = False):
-        self.path = path
-        self.logFile = os.path.join(path, "log.txt")
+    def __init__(
+            self,
+            path: str = None,
+            file: str = None,
+            debug: bool = False):
+        if path is None and file is None:
+            print("DebugLog: No log file or path specified")
```

should raise an exception here, don't let it through.